mandatory/4-tax.js

```diff
@@ -17,7 +21,11 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price) {
+  let taxRate = 20;
+  let priceWithTax = price * (1 + taxRate / 100);
+        return '£'+ priceWithTax.toFixed(2);
+}
```

Comment:
it's better readable if we write `let priceWithTax = price * 1.2` :)
to make this function look ideal you could also fix indents:
```js
function addTaxAndFormatCurrency(price) {
  let taxRate = 20;
  let priceWithTax = price * 1.2;
        return '£'+ priceWithTax.toFixed(2);
  ^^^ too many spaces here before return keyword.
}
```